src/Parser/Core/Modules/CastEfficiency.js

```diff
@@ -9,9 +9,37 @@ import getCastEfficiency from 'Parser/Core/getCastEfficiency';
 import Tab from 'Main/Tab';
 import CastEfficiencyComponent from 'Main/CastEfficiency';
 
+import SPELLS from 'common/SPELLS';
+import ITEMS from 'common/ITEMS';
+
 class CastEfficiency extends Module {
-  static CPM_ABILITIES = [];
-  static SPELL_CATEGORIES = [];
+  static SPELL_CATEGORIES = [
+    {
+      ROTATIONAL: 'Rotational Spell',
+    },
+    {
+      ROTATIONAL_AOE: 'Spell (AOE)',
+    },
+    {
+      COOLDOWNS: 'Cooldown',
+    },
+    {
+      OTHERS: 'Spell',
+    },
+  ];
+
+  static CPM_ABILITIES = [
+    {
```

Comment:
I should probably make it possible to define Cast Efficiency things in modules :stuck_out_tongue: